src/lib/StatsTable.svelte

```diff
@@ -30,21 +31,35 @@
 			<tr>
 				<th>Allocation method</th>
 				{#each Object.values(allocationMethods) as { label }}
-					<th>{label}</th>
+					<th>
+						{#if label==='Grandfathering'}
+							<span class={grandfatheringColor}>{label}</span>
+						{:else}
+							{label}
+						{/if}
+					</th>
 				{/each}
 			</tr>
 		</thead>
 		<tbody>
 			<tr>
 				<th>2030 reductions<br />relative to 2015</th>
 				{#each Object.keys(allocationMethods) as id}
-					<th>{reductions[id][2030] === null ? '-' : $tweenedReductions[id][2030].toFixed(0)}%</th>
+					<th>
+						<span class={id === 'GF' ? grandfatheringColor : ''}>
+							{reductions[id][2030] === null ? '-' : $tweenedReductions[id][2030].toFixed(0)}%
+						</span>
+					</th>
```

Comment:
Changing `th` to `td` fixes the spacing. Also note that you don't need the `span`s. You can directly apply the styling to the table cells:

```suggestion
					<td class={id==='GF' ? grandfatheringColor : ""}>
							{reductions[id][2030] === null ? '-' : $tweenedReductions[id][2030].toFixed(0)}%
					</td>
```

Reply:
> Changing `th` to `td` fixes the spacing. Also note that you don't need the `span`s. You can directly apply the styling to the table cells:

![image](https://github.com/user-attachments/assets/41fba414-ea6a-4869-aad1-cc2bdfe23165)

Now the 2030 and 2040 rows are not aligned anymore it seems @Peter9192